From aee551ac19ccd1fada041e04b8b6e793aaf6d0f4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Mon, 9 Feb 2026 23:46:43 +0100
Subject: [PATCH 123/133] std.Target: update max Linux version to 6.19
X-Developer-Signature: v=1; a=openpgp-sha256; l=781; i=xtex@astrafall.org;
 h=from:subject; bh=wBSEitg2Hr2szd384bV0o8hcak7KPfMy5/pvUbJyotE=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78vjVBI8t2LSje+/3Hld1Vr8fRk4fEqxEKlce3C5+L
 XxuLHt2x0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBU/8lh+GcS/l203DHJYZJn7Z2sxW67jefubBJ07mV5/P/FVuurdU8Y/vumiy5
 XyZz3w/RfbOtPqcwjO6v0eey2Ll/T+e39U2YhJQYA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit fcf0fc225a7e9023d62138e6cbc6d5ffd91453e0)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 72ab18cd0ca4..25cd82fa8cda 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -447,7 +447,7 @@ pub const Os = struct {
 
                                 break :blk default_min;
                             },
-                            .max = .{ .major = 6, .minor = 17, .patch = 0 },
+                            .max = .{ .major = 6, .minor = 19, .patch = 0 },
                         },
                         .glibc = blk: {
                             // For 32-bit targets that traditionally used 32-bit time, we require
-- 
2.52.0

